.github/CONTRIBUTING.md

```diff
@@ -1,3 +1,29 @@
 # Contributing
 
-> TODO
+*Thanks to [IGListKit for writing  a great CONTRIBUTING.md](https://github.com/Instagram/IGListKit/blob/master/.github/CONTRIBUTING.md), upon which our guidelines are based.*
+
+----
+
+We're excited about new contributors and want to make it easy for you to help improve this project. If you run into problems, please open a GitHub issue.
+
+If you want to contribute a fix, or a minor change that doesn't change the API, go ahead and open a pull requests, see details on pull requests below.
+
+If you're interested in making a larger change, we recommend you to open an issue for discussion first. That way we can ensure that the change is within the scope of this project (see our [Vision document](https://github.com/plangrid/ReactiveLists/blob/master/Guides/VISION.md) for details on the scope of the project) before you put a lot of effort into it.
+
+## Pull Requests
+
+1. Fork the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Add an entry to the `CHANGELOG.md` for any breaking changes, enhancements, or bug fixes.
+
+## Issues
+
+We use GitHub issues to track public bugs. Please ensure your description is
+clear and has sufficient instructions to be able to reproduce the issue.
```

Comment:
Can include something to the effect of "Please use the provided Issue Template and ensure your descriptions is ..."